pages/index.js

```diff
@@ -34,23 +41,62 @@ export default function Home() {
       </Head>
 
       <Main>
-        <ul>
-          {cardData.map((card) => {
-            return (
-              <Card
-                key={card.id}
-                name={card.name}
-                onDelete={() => deleteCard(card.id)}
-              />
-            );
-          })}
-        </ul>
+        <div>
+          <Headline>Today</Headline>
+          <HabitForTodayList>
+            {cardData
+              .filter((item) => !item.checked)
+              .map((card) => {
+                return (
+                  <Card
+                    key={card.id}
+                    name={card.name}
+                    onDelete={() => deleteCard(card.id)}
+                    onToggle={() => toggleCard(card.id)}
+                  />
+                );
+              })}
+          </HabitForTodayList>
+          <br />
+          <Headline>Done</Headline>
+          <HabitDone>
+            {cardData
+              .filter((item) => item.checked)
+              .map((card) => {
+                return (
+                  <Card
+                    key={card.id}
+                    name={card.name}
+                    onDelete={() => deleteCard(card.id)}
+                    onToggle={() => toggleCard(card.id)}
+                  />
+                );
+              })}
+          </HabitDone>
+        </div>
         <CreateNewForm onAddNewData={appendCard} />
       </Main>
-    </div>
+    </Wrap>
   );
 }
 
 const Main = styled.main`
-  background-color: white;
+  margin: 0;
+`;
+
+const Wrap = styled.div`
```

Comment:
it looks unusual to have a div around the Header and the Main. Could you transfer these settings to the GlobalStyle.js?

Reply:
Hey thank you @AHen-C0de  i was playing with the background image but forgot to remove the div.